.changeset/witty-buses-worry.md

```diff
@@ -0,0 +1,7 @@
+---
+"@eth-optimism/batch-submitter": minor
+"@eth-optimism/core-utils": patch
+---
+
+- Use raw transaction in batch submitter -- incompatible with L2Geth v0.1.2.1
```

Comment:
What is geth 0.1.2.1? Is it deployed / being used anywhere right now?

Reply:
Ah it is the version of Geth that we were running before our last regenesis ~2/3 weeks ago. This should not be breaking Kovan or Mainnet now that we've got `rawTransaction` in mainnet